src/ncp/ncp_base.hpp

```diff
@@ -575,10 +579,12 @@ class NcpBase
 #endif
     ThreadError InsertPropertyHandler_IPV6_ADDRESS_TABLE(uint8_t header, spinel_prop_key_t key,
                                                          const uint8_t *value_ptr, uint16_t value_len);
+#if OPENTHREAD_FTD
     ThreadError InsertPropertyHandler_THREAD_LOCAL_ROUTES(uint8_t header, spinel_prop_key_t key,
```

Comment:
Very happy to see the reduction in the code size with this change.  Thanks for taking care of this. :+1: 

IIRC, an MTD can still add external route or an on-mesh prefix.  My suggestion is to use a different specific config option to enable/disable these feature and use that instead of using  `#if OPENTHREAD_FTD`.